packages/core/src/html/codeblockProcessor.ts

```diff
@@ -135,20 +151,16 @@ function highlightCodeBlock(node) {
  * @param node the code block element, which is the 'code' node
  * @param showCodeLineNumbers true if line numbers should be shown, false otherwise
  */
-function setCodeLineNumbers(node, showCodeLineNumbers) {
-  const existingClass = node.attribs.class || '';
+export function setCodeLineNumbers(node: DomElement, showCodeLineNumbers: boolean) {
+  const existingClass = (node.attribs && node.attribs.class) ? node.attribs.class : '';
```

Comment:
```suggestion
  const existingClass = node.attribs?.class || '';
```

I think this should have the same effect!